static/bidder-info/robustApps.yaml

```diff
@@ -0,0 +1,9 @@
+aliasOf: "xeworks"
+endpoint: "http://rtb.rbstsystems.live?pid={{.SourceId}}&host={{.Host}}&pbs=1"
+maintainer:
+  email: "prebid@robust-apps.com"

```

Comment:
We've sent an email to this address. Please respond to the email with "received" to confirm that it is reachable.